examples/jobs/httpd.yaml

```diff
@@ -0,0 +1,15 @@
+name: httpd
+containers:
+- image: centos/httpd
+services:
+- name: httpd
+  type: NodePort
+  ports:
+  - port: 8080
+jobs:
+- containers:
```

Comment:
Because we have two different formats. Containers as a subkey and containers as a root key. Depending on whether it's a service or any other kind of `kind` such as jobs, daemonset, statefulset, etc.